src/api/replication_api.js

```diff
@@ -18,27 +18,25 @@ module.exports = {
             method: 'POST',
             params: {
                 type: 'object',
-                required: ['src_bucket_name', 'dst_bucket_name', 'keys'],
+                required: ['src_bucket_name', 'dst_bucket_name', 'objects'],
                 properties: {
                     copy_type: {
                         type: 'string',
                         enum: ['MIX', 'AWS', 'AZURE', 'NB']
                     },
                     src_bucket_name: { $ref: 'common_api#/definitions/bucket_name' },
                     dst_bucket_name: { $ref: 'common_api#/definitions/bucket_name' },
-                    keys: {
-                        type: 'array',
-                        items: {
-                            type: 'string'
-                        }
+                    objects: {
```

Comment:
maybe we should construct better the schema here to the actual structure of this object